src/components/H5PElement/H5PElement.jsx

```diff
@@ -49,9 +54,14 @@ class H5PElement extends Component {
     // Currently, we need to strip oembed part of H5P-url to support NDLA proxy oembed service
     const { oembed_url: oembedUrl } = event.data;
     const url = oembedUrl.match(/url=([^&]*)/)[0].replace('url=', '');
-    const metadata = await fetchH5PTitle(event.data.embed_id);
-    const title = metadata ? metadata.h5p.title : null;
-    onSelect({ url, title });
+    try {
+      const metadata = await fetchH5PMetadata(event.data.embed_id);
+      const title = metadata.h5p.title;
+      onSelect({ url, title });
+    } catch (e) {
+      onSelect({ url });
+      handleError(e);
+    }
```

Comment:
Jeg var litt usikker på konvensjonen med bruk av let/const og null-variabler, men regner med at det beste er å unngå det så langt som det går?

Reply:
Litt usikker på hva du mener, men det ser bra ut som det er nå

Reply:
Nei bare om jeg skulle hatt let-variabler for title og metadata utenfor try/catch, og så kjørt onSelect utenfor så man slipper å kalle på den to ganger

Reply:
Nei, synes det ser bedre ut som du har gjort det nå

Reply:
For å svare på spørsmålet ditt så bruker man som oftest const i javascript/ React. Om det er noe som skal endres på så kan man ofte legge det til komponentens state og oppdatere det slik. https://reactjs.org/docs/state-and-lifecycle.html

Null vs undefined er en evig debatt: https://knowitobjectnet.slack.com/archives/C18NK1F2N/p1579697711031400